editor/src/components/canvas/canvas-strategies/relative-move-strategy.tsx

```diff
@@ -0,0 +1,144 @@
+    return metadata.specialSizeMeasurements.position === 'relative'
+  },
+
+  controlsToRender: [
+    {
+      control: ParentOutlines,
+      key: 'parent-outlines-control',
+      show: 'visible-only-while-active',
+    },
+    {
+      control: ParentBounds,
+      key: 'parent-bounds-control',
+      show: 'visible-only-while-active',
+    },
+  ],
+
+  fitness: (canvasState, interactionState, _sessionState) => {
+    const { interactionData, activeControl } = interactionState
+    if (!(interactionData.type === 'DRAG' && interactionData.drag != null)) {
+      return 0
+    }
+    if (activeControl.type !== 'BOUNDING_AREA') {
+      return 0
+    }
+
+    const selectedElements = getTargetPathsFromInteractionTarget(canvasState.interactionTarget)
+    if (selectedElements.length === 0) {
+      return 0
+    }
+    const filteredSelectedElements = getDragTargets(selectedElements)
+    const last = filteredSelectedElements[filteredSelectedElements.length - 1]
+    const metadata = MetadataUtils.findElementByElementPath(interactionState.latestMetadata, last)
+    if (!metadata) {
+      return 0
+    }
+    const offsets = getStyleOffsets(metadata)
+    if (!offsets) {
+      return 0
+    }
+
+    const hasOffsets =
+      offsets.left != null || offsets.top != null || offsets.right != null || offsets.bottom != null
+
+    return hasOffsets
+      ? 4 // +1 than reorder flow
+      : 1 // there should be a more structured way to define priorities (:
+  },
+
+  apply: (canvasState, interactionState, sessionState) => {
+    const { interactionData } = interactionState
+    if (!(interactionData.type === 'DRAG' && interactionData.drag != null)) {
+      return emptyStrategyApplicationResult
+    }
+
+    const selectedElements = getTargetPathsFromInteractionTarget(canvasState.interactionTarget)
+    if (selectedElements.length === 0) {
+      return emptyStrategyApplicationResult
+    }
+
+    const filteredSelectedElements = getDragTargets(selectedElements)
+    const last = filteredSelectedElements[filteredSelectedElements.length - 1]
+    const metadata = MetadataUtils.findElementByElementPath(sessionState.startingMetadata, last)
+    if (metadata == null) {
+      return emptyStrategyApplicationResult
+    }
+    const offsets = getStyleOffsets(metadata)
+    if (!offsets) {
+      return emptyStrategyApplicationResult
+    }
+
+    return applyMoveCommon(
+      canvasState,
+      interactionState,
+      sessionState,
+      getAdjustMoveCommands(canvasState, interactionState, sessionState),
+    )
+  },
+}
+
+const getStyleOffsets = (meta: ElementInstanceMetadata) => {
+  const getOffsetPropValue = (
+    name: 'left' | 'top' | 'right' | 'bottom',
+    attrs: PropsOrJSXAttributes,
+  ): number | null => {
+    return foldEither(
+      (_) => null,
+      (v) => (v != null ? v.value : null),
+      getLayoutProperty(name, attrs, ['style']),
+    )
+  }
+
+  if (!isRight(meta.element)) {
```

Comment:
```suggestion
  if (isLeft(meta.element)) {
```

Reply:
you're absolutely _right_ (ba-dum-tss) 🏅